core/domain/prod_validators.py

```diff
@@ -1148,9 +1112,6 @@ def _get_external_id_relationships(cls, item):
             base_model_validators.ExternalModelFetcherDetails(
                 'exploration_ids',
                 exp_models.ExplorationModel, [item.id]),
-            # TODO (#10828): Remove validation for cloned_from
-            # exp ids after the field is entirely removed from
-            # all models.
```

Comment:
Why this change?

Reply:
I did not mean to change this. My guess is that I saved a local (older) version of this file after rebasing on develop. It looks like I do not have the changes from #10821. Probably, this is the case for all the below changes, because I did not make any of those either. I will manually merge. 

Thank you so much!

Reply:
Done